src/control/server/server_utils.go

```diff
@@ -268,6 +269,12 @@ func updateMemValues(srv *server, ei *EngineInstance, getHugePageInfo common.Get
 
 	// Fail if free hugepage mem is not enough to sustain average I/O workload (~1GB).
 	if memSizeFreeMb < memSizeReqMb {
+		out, err := exec.Command("/bin/cat /proc/meminfo").Output()
+		if err != nil {
+			srv.log.Errorf("command failed: %s", err.Error())
+		}
+		srv.log.Errorf("meminfo: %s", out)
```

Comment:
Instead of relying on shelling out, I'd suggest opening and reading from the file.